WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationWizardContent.swift

```diff
@@ -0,0 +1,159 @@
+import UIKit
+
+final class SiteInformationWizardContent: UIViewController {
+    fileprivate enum Rows: Int, CaseIterable {
```

Comment:
Could this be `private`?

Reply:
As far as I know, it couldn't, if we want to consume it from the extensions. Will check

Reply:
Turns it, it can! TIL...